src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -74,6 +74,16 @@
   <strong>{{ actionDescription }}</strong> if you proceed.</ng-container>
 </ng-template>
 
+<ng-template #flagsTpl
+             let-value="value">
+  <div *ngFor="let flag of clusterWideFlags;">
+    <span class="badge badge-hdd">{{ flag }}</span>
+  </div>
+  <div *ngFor="let flag of filterIndivFlags(value);">
+    <span class="badge badge-info">{{ flag }}</span>
+  </div>
+</ng-template>
```

Comment:
Should we add a tooltip on each badge telling what it refers to?
just by looking at it I don't know which are cluster-wide or individual flags

Reply:
Yes, I thought about it as well... I haven't had a good idea how to do this.

Reply:
![Screenshot_2020-09-04_13-20-29](https://user-images.githubusercontent.com/8761082/92234020-98e52e80-eeb1-11ea-8393-1251a88ad15d.png)

Reply:
Sorry for the late comment: instead of adding the tooltip to each specific label, which can increase the noise in the UI when moving the mouse over it and it's not explaining the label contents (noout, noin, nodown) but its scope (individual, cluster-wide, ...), why not simply add a clickable tooltip legend via an :information_source: icon in the "**Flags**" column header?

In case of adding tooltips to every flag label, I'd rather use that to explain what "noout, noin, no*" means (but I'm not sure if we have to put tooltips to everything).

Reply:
> Sorry for the late comment: instead of adding the tooltip to each specific label, which can increase the noise in the UI when moving the mouse over it and it's not explaining the label contents (noout, noin, nodown) but its scope (individual, cluster-wide, ...), why not simply add a clickable tooltip legend via an  icon in the "**Flags**" column header?

Yes, sure, I'll check how this can be done.

> In case of adding tooltips to every flag label, I'd rather use that to explain what "noout, noin, no*" means (but I'm not sure if we have to put tooltips to everything).

No, I don't think we have to ;)

Reply:
We agreed on implementing it in a follow-up PR: https://tracker.ceph.com/issues/47406

Reply:
thanks @s0nea !

Reply:
BTW, this Patternfly layout could help on that one (popover titles on column headings):

![image](https://user-images.githubusercontent.com/37327689/93319647-6a037c80-f810-11ea-866a-908fe4185803.png)